product_total_weight_from_packaging/tests/test_product_total_weight_from_packaging.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo.tests import TransactionCase
+
+
+class TestProductTotalWeightFromPackaging(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

Comment:
Shouldn't this extend `TestProductCommon` instead ? Then no need for tracking_disable

```suggestion
from odoo.addons.product.tests.common import TestProductCommon


class TestProductTotalWeightFromPackaging(TestProductCommon):
    @classmethod
    def setUpClass(cls):
        super().setUpClass()
```